www/convertUtils.js

```diff
@@ -28,10 +28,19 @@ module.exports = {
     */
     toCordovaFormat: function (contact) {
         var value = contact.birthday;
-        try {
-          contact.birthday = new Date(parseFloat(value));
-        } catch (exception){
-          console.log("Cordova Contact toCordovaFormat error: exception creating date.");
+        if (value !== null) {
```

Comment:
I did that null check to skip the part where we enter the try-catch block and try to create a new Date. I think this has impact on performance, it's faster.